gammapy/data/event_list.py

```diff
@@ -654,6 +473,232 @@ def plot_time_map(self, ax=None):
+
+        ax.invert_xaxis()
+        ax.grid()
+
+        divider = make_axes_locatable(ax)
+        cax = divider.append_axes("right", size="5%", pad=0.05)
+        plt.colorbar(im, cax=cax)
+
+    def plot_image(self, ax=None, number_bins=50):
+        """Plot the counts as a function of x and y camera coordinate.
+
+        TODO: fix the histogramming ... this example shows that it's currently incorrect:
+        gammapy-data-show ~/work/hess-host-analyses/hap-hd-example-files/run023000-023199/run023037/hess_events_023037.fits.gz events -p
+        Maybe we can use the FOVCube class for this with one energy bin.
+        Or add a separate FOVImage class.
+        """
+        import matplotlib.pyplot as plt
+        ax = plt.gca() if ax is None else ax
+
+        max_x = max(self.table['DETX'])
+        min_x = min(self.table['DETX'])
+        max_y = max(self.table['DETY'])
+        min_y = min(self.table['DETY'])
+
+        x_edges = np.linspace(min_x, max_x, number_bins)
+        y_edges = np.linspace(min_y, max_y, number_bins)
+
+        count_image, x_edges, y_edges = np.histogram2d(
+            self.table[:]['DETY'], self.table[:]['DETX'],
+            bins=(x_edges, y_edges)
+        )
+
+        ax.set_title('# Photons')
+
+        ax.set_xlabel('x / deg')
+        ax.set_ylabel('y / deg')
+        ax.imshow(count_image, interpolation='nearest', origin='low',
+                  extent=[x_edges[0], x_edges[-1], y_edges[0], y_edges[-1]])
+
+    def plot_offset_hist(self, ax=None):
+        """Plot counts as a function of camera offset."""
+        raise NotImplementedError
+
+    def plot_energy_offset(self, ax=None):
+        """Plot energy dependence as a function of camera offset."""
+        raise NotImplementedError
+
+
+class EventListLAT(EventListBase):
+    """Event list for IACT dataset
+
+    To load an example Fermi-LAT event list (the one corresponding to the 2FHL catalog dataset):
+
+    >>> filename = '$GAMMAPY_EXTRA/datasets/fermi_2fhl/2fhl_events.fits.gz'
+    >>> events = EventList.read(filename)
+    """
+    def __init__(self, table):
+        EventListBase.__init__(self, table)
+
+    def plot_image(self, center, size):
+        """A quick look function to generate a count skymap with all the photons
+        within a certain square.
+        Fermi-LAT eventlist could encompass large fraction of the sky,
+        this way we can restirct the skymap to a squared region of interest (ROI)
+
+        Parameters:
+        -----------
+        center : `~astropy.coordinates.SkyCoord`
+            Sky circle center
+        size : `~astropy.coordinates.Quantity`
+            size of the square defining our ROI
+        """
+        ref_image = SkyImage.empty(
+            nxpix=400, nypix=400, binsz=0.02,
+            xref=center.icrs.ra.deg, yref=center.icrs.dec.deg,
+            coordsys='CEL', proj='TAN',
+            )
+        counts_image = SkyImage.empty_like(ref_image)
```

Comment:
Default for SkyImage.empty is to fill data with zeros:
http://docs.gammapy.org/en/latest/api/gammapy.image.SkyImage.html#gammapy.image.SkyImage.empty

So you can just say `image = SkyImage.empty` and then fill this with counts here, no need to have another `SkyImage.empty_like` call.

I would also suggest to just compute `nxpix` and `nypix` directly and to never call `cutout`.